test-feature/wfx_test_target.py

```diff
@@ -120,7 +122,13 @@ def _prepare_test_data(self, parameters):
 
         pds_string = "#include \"" + pds_env['PDS_DEFINITION_ROOT'] + pds_env['PDS_DEFINITION_FILE']\
                      + "\"\n\n" + pds_compatibility_text + pds_sections
-        compressed_string = compress_string(pds_string, self.required_options)
+
+        compressed_string = compress_string(pds_string, format=self.pds_output_format)
+
+        # The TLV format has some binary data that needs to be included
+        if self.pds_output_format == "tlv":
+            compressed_string = repr(compressed_string).strip("'")
```

Comment:
This is to include the non string data from the `tlv` format which I thought was a string but includes some required null bytes.  
Note that `wfx_test_agent` needs to use `echo -e` do decode the `\0x00` 
That's anyway compatible with other pure strings pds formats.